### ISSUE TITLE
No "tefreqs:" line in the out put from the tutorial...

### ISSUE THREAD

**Issue by mjocic56** [stevengj/mpb#68]
Hello,
I started going through the tutorial for "Our first band structure" and I couldn't help but notice that
my output for `ms.run_te()` doesn't have the `tefreqs:` line, that is, my output looks like this:

Initializing eigensolver data
Computing 8 bands with 1e-07 tolerance
16 k-points
  Vector3<0.0, 0.0, 0.0>
  Vector3<0.1, 0.0, 0.0>
  Vector3<0.2, 0.0, 0.0>
  Vector3<0.30000000000000004, 0.0, 0.0>
  Vector3<0.4, 0.0, 0.0>
  Vector3<0.5, 0.0, 0.0>
  Vector3<0.5, 0.1, 0.0>
  Vector3<0.5, 0.2, 0.0>
  Vector3<0.5, 0.30000000000000004, 0.0>
  Vector3<0.5, 0.4, 0.0>
  Vector3<0.5, 0.5, 0.0>
  Vector3<0.4, 0.4, 0.0>
  Vector3<0.3, 0.3, 0.0>
  Vector3<0.19999999999999996, 0.19999999999999996, 0.0>
  Vector3<0.09999999999999998, 0.09999999999999998, 0.0>
  Vector3<0.0, 0.0, 0.0>
elapsed time for initialization: 0.0027337074279785156
Outputting epsilon.h5...
elapsed time for k point: 0.16631245613098145
elapsed time for k point: 0.04373908042907715
elapsed time for k point: 0.24829554557800293
elapsed time for k point: 0.03610086441040039
elapsed time for k point: 0.06210970878601074
elapsed time for k point: 0.055446624755859375
elapsed time for k point: 0.04317760467529297
elapsed time for k point: 0.05084419250488281
elapsed time for k point: 0.04191470146179199
elapsed time for k point: 0.027718544006347656
elapsed time for k point: 0.025140047073364258
elapsed time for k point: 0.027942419052124023
elapsed time for k point: 0.03310561180114746
elapsed time for k point: 0.04081010818481445
elapsed time for k point: 0.060736656188964844
elapsed time for k point: 0.20693731307983398
Band 1 range: 0.0 at Vector3<0.0, 0.0, 0.0> to 0.49683586573979743 at Vector3<0.5, 0.5, 0.0>
Band 2 range: 0.4415884758922276 at Vector3<0.5, 0.0, 0.0> to 0.5931405182467208 at Vector3<0.5, 0.5, 0.0>
Band 3 range: 0.5931535893551052 at Vector3<0.5, 0.5, 0.0> to 0.7732265678658415 at Vector3<0.0, 0.0, 0.0>
Band 4 range: 0.6791690135255335 at Vector3<0.5, 0.5, 0.0> to 0.80968916620506 at Vector3<0.3, 0.3, 0.0>
Band 5 range: 0.8241814492203078 at Vector3<0.5, 0.30000000000000004, 0.0> to 0.922996527814745 at Vector3<0.0, 0.0, 0.0>
Band 6 range: 0.8819770936729684 at Vector3<0.5, 0.5, 0.0> to 1.0291597307232732 at Vector3<0.5, 0.0, 0.0>
Band 7 range: 0.8819818143162569 at Vector3<0.5, 0.5, 0.0> to 1.0860733972501748 at Vector3<0.5, 0.0, 0.0>
Band 8 range: 1.0878690870671277 at Vector3<0.5, 0.0, 0.0> to 1.1142329750945672 at Vector3<0.1, 0.0, 0.0>
Gap from band 2 (0.5931405182467208) to band 3 (0.5931535893551052), 0.0022036876522750046%
Gap from band 4 (0.80968916620506) to band 5 (0.8241814492203078), 1.773981719045087%
Gap from band 7 (1.0860733972501748) to band 8 (1.0878690870671277), 0.16520122587482305%
total elapsed time for run: 1.212172269821167
done

Has the output changed overtime, or my package is not working properly?
I installed it through Conda.

Kind regards

**Comment by ChristopherHogan** [comment]
The regular console output doesn't have this issue, so I assume you are trying to redirect the output like this `python tutorial.py > out.txt`. You can work around this by adding the `-u` flag like this: `python -u tutorial.py > out.txt`,  for unbuffered output.

**Comment by mjocic56** [comment]
I'm trying it in Jupyter notebook, not in console. Is that the issue?

Also I can't seem to load `meep` and `mpb` through regular console since they're tied to Anaconda only, that is I can't execute the program if I go through regular console. I have to start Anaconda and then open a notebook from environment that it's all installed in...

BTW I tried installing `meep` form Ubuntu repo, the install showed no errors but python showed error when loading the module.

**Comment by ChristopherHogan** [comment]
> I'm trying it in Jupyter notebook, not in console. Is that the issue?

Yes, that is the problem. There's an open ipython issue [here](https://github.com/ipython/ipython/issues/1230). The output from C will show up in your terminal if you're serving your own notebook though (the notebook is only showing the python output and not the C extension output). 

> Also I can't seem to load meep and mpb through regular console since they're tied to Anaconda only

You just have to make sure your path is set up correctly and your environment is activated. Instructions can be found [here](https://meep.readthedocs.io/en/latest/Installation/#conda-packages).

> BTW I tried installing meep form Ubuntu repo, the install showed no errors but python showed error when loading the module.

Yes, the current debian package is an older version that doesn't yet include the python interface.

**Comment by mjocic56** [comment]
> You just have to make sure your path is set up correctly and your environment is activated. Instructions can be found here.

Thank you for support.

Kind regards
